Pod/Classes/APMAlertAnimation.swift

```diff
@@ -70,5 +71,6 @@ class APMAlertAnimation: NSObject, UIViewControllerAnimatedTransitioning {
                     finished in
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/almas-dev/APMAlertController/blob/b2d5b91114f6a23c8648778f00f62f7d3e83fecd/Pod/Classes/APMAlertAnimation.swift#L71"></span>Closure parameters should be on the same line as opening brace.</td>
    </tr>
  </tbody>
</table>